packages/rehype-docz/src/index.ts

```diff
@@ -28,11 +28,15 @@ const removePlayground = (code: string) => {
 const addCodeProp = async (node: any, idx: number) => {
   const name = componentName(node.value)
   const tagOpen = new RegExp(`^\\<${name}`)
-
   if (isPlayground(name)) {
     const formatted = await format(nodeToString(node))
     const code = formatted.slice(1, Infinity)
-    const child = strip(removePlayground(code)).trim()
+    const child = strip(removePlayground(code))
+      .trim()
+      .split(`'`)
```

Comment:
I think that is better to use `replace` instead of `join` and `split` in this case, no?!

Reply:
Yeah, I agree. Thanks for doing the change yourself #205 

Cheers !